tests/installation/bootloader_hyperv.pm

```diff
@@ -129,6 +140,9 @@ sub run {
     if (grep { /Microsoft Windows \[Version 6.3.*\]/ } $winver) {
         $winserver = 2012;
     }
+    elsif (grep { /Microsoft Windows \[Version 10.0.*\]/ } $winver) {
```

Comment:
Is there a reason to use grep instead of perl regexp `=~`? Looks like $winver is not a list. WDYT?

Reply:
No particular reason. I'd rather keep it, so I don't have to re-validate in light of no other substantial problems with this PR. If you don't mind...

Reply:
Sure, I though maybe there is a reason I don't see, as there are scenarios where grep calls make sense, e.g. when want to delete matching elements from an array in one-liner.